README.md

```diff
@@ -165,7 +170,7 @@ After initializing the registry, you can pass exception mappers of `(Throwable)
 signature from the `ExceptionMappersStorage` to an `ErrorPresenter`:
 
 ```kotlin
-val alertErrorPresenter = AlertErrorPresenter(
+val alertErrorPresenter = AlertErrorPresenter<StringDesc>(
```

Comment:
but alert not typed...it already StringDesc

Reply:
Fixed.